docs/docs/platform/authentication/email-templates.md

```diff
@@ -1,85 +1,68 @@
 ---
-title: 'Email templates'
+title: 'Email Templates'
 sidebar_position: 4
 ---
 
-The following emails can be sent as part of the authentication flow:
+Nhost Auth sends out transactional emails as part of the authentication service. These emails can be modified using email templates.
 
-- Sign up confirmation email (when using email + password)
-- Reset password email (when using email + password)
-- Passwordless login email (when using Magic Link)
-- Confirm email change (any sign-up method)
+The following email templates are available:
 
----
-
-## Enabling custom email templates
+- **email-verify** - Verify email address
+- **email-confirm-change** - Confirm email change to a new email address.
+- **signin-passwordless** - Magic Link
+- **password-reset** - Reset password
 
-If you have developed custom email templates, you must make them available over HTTP and then point Nhost to them. You can host the templates on your server, or use a public repository on GitHub.
+Changing email templates is only available for apps on the [Pro and Enterprise plan](https://nhost.io/pricing).
 
-Go to **Users -> Login settings** and scroll down to **Custom email templates**, and set the URL to where your templates are located:
+## Update Email Templates
 
-![Email templates](/img/platform/email-templates.svg)
+Your app must be connected to a GitHub repository using the [GitHub Integration](/platform/github-integration) to be able to change the email templates.
 
-You only need to define the base URL to point to your hosted templates. The UI will give you a hint about where Nhost will look for your actual template files.
+Email templates are automatically deployed during a deployment, just like database migrations, Hasura metadata, and Serverless Functions.
 
----
+## File Structure
 
-## File structure
+Emails are located in the `nhost/` folder like this:
 
 The email templates should be provided as body.html and subject.txt files in this predefined folder structure:
 
 ```txt
-// At base URL (e.g. https://yourapp.com/email-templates/)
-en/
-  email-confirm-change/
-    body.html
-    subject.txt
-  email-verify/
-    body.html
-    subject.txt
-  password-reset/
-    body.html
-    subject.txt
-  signin-passwordless/
-    body.html
-    subject.txt
-
-// Other language versions
-fr/
-  /* ... */
-se/
-  /* ... */
+|____nhost       <-- Nhost folder
```

Comment:
Could you please use `└─` here?

Similar to how it's displayed on "Get started with Nhost CLI" page:
```
my-nhost-app/
  └─ nhost/
      ├─ config.yaml
      ├─ emails/
      ├─ metadata/
      ├─ migrations/
      └─ seeds/
```